_examples/bundler/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:stretch
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    cmake \
+    libcurl4-openssl-dev \
+    libicu-dev \
+    libmagic-dev \
+    libmariadbclient-dev \
+    libsqlite3-dev \
+    libsystemd-dev \
+    pkg-config \
+    ruby ruby-dev ruby-bundler \
+    zlib1g-dev
+
+WORKDIR /data
+
+COPY docker-entrypoint /usr/local/bin/docker-entrypoint
+RUN chmod +x /usr/local/bin/docker-entrypoint
+
+ENTRYPOINT ["/usr/local/bin/docker-entrypoint"]
```

Comment:
Any reason why it's `_examples/` and not `examples/`?

Reply:
The `_examples/` directory can act as a temporary directory, it is not necessary for building with mistry. The `go/build` package also ignores these files https://golang.org/pkg/go/build/#Context.Import.